tls_main.c

```diff
@@ -563,6 +567,40 @@ static int do_tls_setsockopt_conf(struct sock *sk, char __user *optval,
 	return rc;
 }
 
+#define TLS_LUA 99
+static DEFINE_MUTEX(mtx);
+static lua_State* L;
+#define raise_err(msg) pr_warn("[lua] %s - %s\n", __func__, msg);
+
+static int do_tls_lua(char __user *optval, unsigned int optlen)
```

Comment:
Code style doesn't follow already existing code.
This is bad. You should always follow the existing code's style.